### PR TITLE
feat: add loader

### PR DESCRIPTION
**Pull request by Kovalenko77**

### CODE REVIEW

File: src/Comments.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { useState, useEffect } from 'react';
 import Grid from '@mui/material/Grid';
 import CardComment from './CardComment';
-
+import Loader from './Loader';
```
Comment (romanalesenkov1) [inline src/Comments.tsx:5]: add a line break between imports and interface

File: src/Comments.tsx [hunk 2]
```diff
@@ -14,23 +14,38 @@ interface CommentsResponse {
 
 const Comments = () => {
   const [comments, setComments] = useState<Comment[]>([]);
+  const [isDataLoading, setIsDataLoading] = useState(false);
```
Comment (romanalesenkov1) [inline src/Comments.tsx]: probably the initial value should be true?
```suggestion
  const [isDataLoading, setIsDataLoading] = useState(true);
```

File: src/Comments.tsx [hunk 3]
```diff
@@ -14,23 +14,38 @@ interface CommentsResponse {
...
   }, []);
 
   return (
-    <Grid className="card-wrapper" container spacing={2}>
-      {comments.map((item, index) => (
-        <Grid item xs={12} sm={6} md={3} key={comments.indexOf(item)}>
-          <CardComment key={index} title={item.Title} content={item.Content} />
+    <>
+      {isDataLoading && <Loader />}
+      {
```
Comment (romanalesenkov1) [inline src/Comments.tsx]: do you need these brackets?

File: src/Comments.tsx (hunk 1 above)
Comment (Kovalenko77) [reply src/Comments.tsx:5]: Done. Thanks.

File: src/Comments.tsx (hunk 2 above)
Comment (Kovalenko77) [reply src/Comments.tsx]: Yes. Thanks.

File: src/Comments.tsx (hunk 3 above)
Comment (Kovalenko77) [reply src/Comments.tsx]: No. Thanks.

**Pull request merged**